Google Scholar.js

```diff
@@ -1,5 +1,6 @@
 {
 	"translatorID": "57a00950-f0d1-4b41-b6ba-44ff0fc30289",
+	"translatorType": 4,
```

Comment:
Still need to fix this. How is this even happening though? Are you using Scaffold?